docs/conf.py

```diff
@@ -352,7 +352,7 @@ def force_gc(gallery_conf, fname):
     "gallery_dirs": gallery_dirs,
     "subsection_order": subsection_order,
     "filename_pattern": os.environ.get("TVM_TUTORIAL_EXEC_PATTERN", ".py"),
-    "find_mayavi_figures": False,
+    "image_scrapers": ("mayavi",),
```

Comment:
This should be equivalent on recent versions of Sphinx: https://github.com/sphinx-gallery/sphinx-gallery/blob/master/sphinx_gallery/gen_gallery.py#L150-L156

Reply:
just curious, what was the problem being solved by adding this here? right now we're mostly worried about the Sphinx version pinned in ci-gpu, so while I think it's good for us to support a more recent sphinx, we need to dig ourselves out of relying on ci-gpu for all docs purposes (e.g. and use it only when doing a full docs build including gpu-dependent galleries) before we should think about making the docs flow work on a modern ubuntu.

Reply:
just laziness, initially I was using a more recent version of Sphinx that needed this fix to run but with using the versions from ci-gpu this isn't necessary